datasetinsights/evaluation_metrics/metrics_utils.py

```diff
@@ -0,0 +1,47 @@
+import collections
+
+import numpy as np
+
+
+def mean_metrics_over_iou(metric_per_iou):
+    """Calculate mean value over IOUs.
+
+    Args:
+        metric_per_iou (dict): metric records for each iou
+
+    Returns (float):
+        mean metric values over IOUs
+    """
+    mean_sum = 0
+    for mean_ap in metric_per_iou:
+        result = mean_ap.compute()
+        mean_sum += np.mean(
+            [result_per_label for result_per_label in result.values()]
+        )
+    return mean_sum / len(metric_per_iou)
+
+
+def filter_pred_bboxes(pred_bboxes, max_detections):
+    """Save bboxes with same label in to a dictionary.
+
+    This operation only apply to predictions for a single image.
```

Comment:
I am confused by what this method tries to achieve. The input is a list of bounding box object, and the output is also a list of bounding boxes, right? So why is the concept `labels` applies to this method?